controllers/loadtest_controller.go

```diff
@@ -124,11 +141,13 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return ctrl.Result{}, nil
 }
 
-// checkMissingPods attempts to check if any pods is missing for current
-// loadtest.
-func checkMissingPods(currentLoadTest *grpcv1.LoadTest, allRunningPods *corev1.PodList) *grpcv1.LoadTestMissing {
+// checkMissingPods attempts to check if any required component is missing from
+// the current load test. It takes reference of the current load test and a pod
+// list that contains all running pods at the moment returns all missing
```

Comment:
nit: this sentence is a little complex, perhaps adding a command and switching to a gerund may make it a little more readable.

```suggestion
// list that contains all running pods at the moment, returning all missing
```